korman/properties/modifiers/water.py

```diff
@@ -421,10 +421,10 @@ def register(cls):
                                        description="Largest wave length",
                                        min=0.1, max=50.0,
                                        default=cls._max_length_default)
-        cls.amplitude = IntProperty(name="Amplitude",
+        cls.amplitude = FloatProperty(name="Amplitude",
                                     description="Multiplier for wave height",
-                                    subtype="PERCENTAGE",
-                                    min=0, max=100,
+                                    precision=3,
+                                    min=0.001, max=1.000, # Minimum must be above zero for shores to work!
```

Comment:
Will this require Age creators to fix their amplitude values when they upgrade Korman?

Reply:
Things seemed to carry over fine on both my Noloben and Laki blends, so no, I don't believe so

Reply:
And yeah, I was thinking about the clamp for when the shores mod is active, but was not entirely sure how to do that. The UI scripts are hit or miss when I modify them. 😊